### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.4 - es. 3.36 (14 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#295] [Esercizi M-Z]
![es.36](https://github.com/user-attachments/assets/b4f2d139-10d4-4322-9be8-d9c99aac1ddf)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/18160e3d-a1c0-40c4-8c01-979ebaabbd0f)

**Comment by SickRickX2** [comment]
![immagine](https://github.com/user-attachments/assets/65fef6e1-21da-4e88-a80b-cb7b42888d18)

**Comment by scuty2000** [comment]
![image](https://github.com/user-attachments/assets/31bf7a89-49fd-4862-9d89-132160cbe27c)
